hyppo/kgof/kernel.py

```diff
@@ -13,6 +10,8 @@
 import autograd
 import autograd.numpy as np
 
+from ..tools import compute_kern
+
 
 class Kernel(ABC):
```

Comment:
I played around with `compute_kern` a lot, the issue ended up being that the way it's written in hyppo doesn't handle the ArrayBox values that `autograd` returns well and throws an error because it doesn't know what to do with them. Further, I tried extracting just the numpy values that are within the ArrayBox object and feed that to `compute_kern`, but I had no luck -- I think it's going to have to stay this way for it to work.

Reply:
Why ArrayBox and not numpy arrays?

Reply:
The module still accepts and works with `numpy` arrays end-to-end, there's just an intermediate conversion to ArrayBox here when I use `autograd` for kernel computations and operations before it comes back to usable `numpy` values.

Reply:
ok can you leave a `TODO` comment to change this in the future? And make an issue to change it? Then we can resolve this

Reply:
Sounds good, taking care of it right now.

Reply:
Issue created and T0D0 added!